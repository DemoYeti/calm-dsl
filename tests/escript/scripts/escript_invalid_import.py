# python3;failure

from bs4 import BeautifulSoup
URL = "https://realpython.github.io/fake-jobs/"
page = urlreq(url, verb='GET')
soup = BeautifulSoup(page.content, "html.parser")