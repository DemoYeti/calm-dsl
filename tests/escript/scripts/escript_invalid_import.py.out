Error in script: import of 'bs4' is unauthorized
